hobo_fields.gemspec

```diff
@@ -1,5 +1,5 @@
 name = File.basename( __FILE__, '.gemspec' )
-version = File.read(File.expand_path('../VERSION', __FILE__)).strip
+version = File.read(File.expand_path('VERSION', __dir__)).strip
```

Comment:
do we also need to update the gemspec to push the gem to gemfury for the major version bump that we're doing?

Reply:
Yes, I'm doing that next.